cyences_app_for_splunk/default/macros.conf

```diff
@@ -710,7 +709,7 @@ definition = ".ad.crossrealms.com, .crossrealms.com, .local"
 
 # User Inventory
 [cs_user_inventory_user_postfixes]
-definition = ".ad.crossrealms.com, @crossrealms.com, .crossrealms.com, .local"
+definition = ".ad.crossrealms.com, @crossrealms.com, @CROSSREALMS.LOCAL, .crossrealms.com, .local"
```

Comment:
Is this case insensitive, right? Just confirming.

Reply:
Yes, everything matching after converting it to lower case.